### PR TITLE
Manual name entry

### PR DESCRIPTION
**Pull request by ashfurrow**
This is work for [PURCHASE-640](https://artsyproduct.atlassian.net/browse/PURCHASE-640) and requires users who skip the credit card swipe to manually enter in their name. ~It's a WIP because it's not done yet.~

### CODE REVIEW

File: Kiosk/Bid Fulfillment/Models/RegistrationCoordinator.swift [hunk 1]
```diff
@@ -32,32 +34,34 @@ class RegistrationCoordinator: NSObject {
...
-
         case .confirmVC:
             return storyboard.viewController(withID: .RegisterConfirm)
         }
     }
 
     func nextViewControllerForBidDetails(_ details: BidDetails, sale: Sale) -> UIViewController {
+        if (sale.bypassCreditCardRequirement) {
+            if notSet(details.newUser.name.value) {
+                return viewControllerForIndex(.nameVC)
```
Comment (ashfurrow) [inline Kiosk/Bid Fulfillment/Models/RegistrationCoordinator.swift]: This isn't working yet.

**Comment by ashfurrow** [comment]
Okay, this is working, I just want some test coverage...

**Comment by ashfurrow** [comment]
Here's what our storyboard looks like:

screen shot 2019-01-17 at 5 47 33 pm

@briansw I was thinking we might take this opportunity to clean up the capitalization in this UI, what do you think?

**Comment by briansw** [comment]
Yeah that'd be great. We use sentence case.
Ex: Enter your full name

**Comment by ashfurrow** [comment]
@briansw okay thanks, I'll take care of that.

**Comment by ashfurrow** [comment]
Hmm, tests passing locally but failing on CI 😞

**Comment by ashfurrow** [comment]
Looks like it's not accessing the correct fonts pod.

**Comment by ashfurrow** [comment]
Finally green! Okay, feel free to review+merge @orta and maybe I'll get the copy changes in this PR. Or next time.

File: KioskTests/Bid Fulfillment/SwipeCreditCardViewControllerTests.swift [hunk 2]
```diff
@@ -25,7 +25,8 @@ class StubKeys: EidolonKeys {
 
 class SwipeCreditCardViewControllerTests: QuickSpec {
     override func spec() {
-        it("unbinds bidDetails on viewWillDisappear:") {
+        // This opens a microphone access dialogue in macOS from the simulator, so let's skip it for now.
+        pending("unbinds bidDetails on viewWillDisappear:") {
```
Comment (orta) [inline KioskTests/Bid Fulfillment/SwipeCreditCardViewControllerTests.swift]: good call

**Comment by orta** [comment]
Yep, cool, this makes sense - just a reminder to you that this PR lands into the `xcode9` branch  and not master 👍

**Comment by ashfurrow** [comment]
Cool thanks 👍

**Pull request merged**